scripts/build-depends.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
+from email.parser import Parser
+with open('debian/control') as fp:
+    headers = Parser().parse(fp)
+    packages = [val.strip().split(' ')[0]
+                for val in headers['Build-Depends'].split(',')]
```

Comment:
the previous implementation excluded debhelper-compat, is that still needed?

Reply:
I don't think so, apt seems to do the right thing.